python/pytinydiffsim_includes.h

```diff
@@ -121,9 +121,9 @@ inline MyAlgebra::VectorX MyInverseKinematics(tds::MultiBody<MyAlgebra>& mb, int
 {
     MyAlgebra::VectorX q_target;
 #ifdef USE_IK_JAC_TRANSPOSE
-    TINY::TinyInverseKinematics<MyScalar, MyTinyConstants, TINY::IK_JAC_TRANSPOSE> inverse_kinematics;
+    TINY::TinyInverseKinematics<MyScalar, MyTinyConstants, MyAlgebra, TINY::IK_JAC_TRANSPOSE> inverse_kinematics;
```

Comment:
Haven't looked in details but: do we need to pass the MyScalar and MyTinyConstants argument, if we pass MyAlgebra? Can't we use MyAlgebra::Scalar and MyAlgebra::Constants (or something like that)? For example, tds::MultiBody only takes one argument.

Reply:
Fixed in fecf817